integration_tests/test_c_interop_03.py

```diff
@@ -0,0 +1,14 @@
+from ltypes import ccall, f32, f64, i32, i64, CPtr, pointer, Pointer, p_c_pointer
+
+@ccall
+def f_pi32_i32(x: CPtr) -> i32:
+    pass
+
+def test_c_callbacks():
+    xi32: i32
+    xi32 = 3
+    p: CPtr
+#    p_c_pointer(p, xi32)
+#    assert f_pi32_i32(p) == 4
```

Comment:
@czgdp1807 I think we need to get this one working. The `p_c_pointer` would behave like CLoc. Then we'll be able to call C functions that accept a pointer, using CPtr as the declaration.

Reply:
Fixed in #587.